src/View/TemplatePaths.php

```diff
@@ -616,12 +616,13 @@ public function getTemplateSource($controller = 'Default', $action = 'Default')
      * <PackageKey>_<SubPackageKey>_<ControllerName>_<prefix>_<SHA1>
      * The SH1 hash is a checksum that is based on the file path and last modification date
      *
-     * @param string $pathAndFilename
+     * @param string|null $pathAndFilename
      * @param string $prefix
      * @return string
      */
     protected function createIdentifierForFile($pathAndFilename, $prefix)
```

Comment:
For the reference: this is called in `getTemplateIdentifier()` with the result of `resolveTemplateFileForControllerAndActionAndFormat()` which can be `string|null`.